source/includes/api/_dataset.md

```diff
@@ -161,16 +222,13 @@ curl -X GET https://api.resourcewatch.org/v1/dataset?sort=-name,+description
 curl -X GET https://api.resourcewatch.org/v1/dataset?sort=user.role
 ```
 
-#### Special sorting criteria
+The API currently supports sorting by means of the `sort` query parameter. Sorting can be done using any field from the dataset data model, as well as `user.name` and `user.role` (sorting by user data is restricted to ADMIN users). Sorting by nested fields is not supported at the moment.
 
-There are some criteria for special sorting in datasets:
+Multiple sorting criteria can be used, separating them by commas.
 
-- `metadata`: delegates sorting to the metadata component, sorting by the name field of the metadata.
-- `most-viewed` delegates sorting to the graph component, sorting by the datasets that have been queried more frequently. Supports ascending/descending order.
-- `most-favorited`: delegates sorting to the graph component, sorting by the datasets that have been more favorited. Supports ascending/descending order.
-- `relevance`: delegates sorting to the metadata component, sorting by the datasets which metadata better match the search criteria. Can only be used in conjunction with a `search` parameter. Does not support ascending order.
+You can also specify the sorting order by prepending the criteria with either `-` or `+`. By default, `asc` order is assumed.
 
-Special search criteria must be used as sole sorting criteria, as it's not possible to combine any of them with any other search criteria.
+#### Special sorting criteria
 
 > Sorting datasets with special criteria
 
```

Comment:
In the example, should be `most-favorited`, not `-most-favorited`, right? If not, this needs explanation.

Reply:
both are valid, this will be clarified in my upcoming changes